client/src/app/admin/events/page.tsx

```diff
@@ -0,0 +1,11 @@
+"use client"
+
+import { AdminHeading } from "../AdminHeading"
+
+export default function AdminEventsPage() {
+  return (
+    <>
+      <AdminHeading title="Events" />
```

Comment:
can we make something saying its still in progress? client might complain theres a bug lol

Reply:
Added the work in progress screen (similar to the one thats currently in the events page)